backend/src/routes/index.js

```diff
@@ -23,6 +23,17 @@ async function ensureUser(req, res, next) {
   next()
 }
 
+async function checkSingularity({userId, computerId, sessionId}) {
```

Comment:
We should rename this to `fetchUserIdsBySingularities` to better reflect the job this method does.